src/MultiTypeSymbolTable.chpl

```diff
@@ -62,10 +62,15 @@ module MultiTypeSymbolTable
                                      "Registering symbol: %s ".format(userDefinedName));            
             }
             
+            // RE: Issue#729 we no longer support multiple name registration of the same object
+            if (registry.contains(name)) {
+                registry -= name;
+            }
+
```

Comment:
Dev Note: With in-place modification on the client side, it was necessary to remove the "old name" to handle the case where a user does something like this:
```
pda.register("ok_name")
pda.register("better_name")
```

Reply:
nice catch!